src/js/components/landing/recentActivity/SubmissionStatus.jsx

```diff
@@ -63,11 +63,15 @@ export class SubmissionStatus extends React.Component {
 	render() {
 
 		const colors = this.progressBar(this.props.status);
+		let label = this.statusStrings[this.props.status]
```

Comment:
Did you check what the wording should be for the status on these? I seem to remember discussing how it should just say "Updated" but we should check.